gitpod-io/gitpod
# Truncate branch name in prebuilds tab, and more

## Description

https://user-images.githubusercontent.com/462705/143479443-5f0a6714-13c8-4216-bb70-bdcff31149e5.mov

This does a couple things:

- Fixes #5332 by truncating the branch name in the prebuilds tab
- Adds a title to both the branch name and the commit (which was already truncated) so that the user is still able to read the complete info even when truncated. See the attached short video.
- Aligns the branch name vertically to the rest of the info. Something that was bugging me to no end because I'm both border-line OCD and 45% UX designer 🤪. It's done with retrocompatibility in mind, to avoid a refactor. I'm willing to do the refactor if you want me to, tho.
- Updates the README with some info from the awesome @iQQBot , and a screenshot.

## Related Issue(s)

Fixes #5332

## How to test


Just start the dashboard and navigate to the prebuild tab of a project. Make sure to have the prebuild of a branch with a long name so that the actual truncation can happen.

## Release Notes

```release-note
Tweaks to the UI of the Prebuilds tab in the dashboard
Some improvements to the README of the dashboard, aimed at contributors
```

## Documentation
